eng/Generate.ps1

```diff
@@ -5,7 +5,7 @@ $ErrorActionPreference = 'Stop'
 
 function Invoke-AutoRest($baseOutput, $title, $autoRestArguments)
 {
-    $command = "npx autorest-beta $script:debugFlags $autoRestArguments"
+    $command = "npx  @autorest/autorest $script:debugFlags $autoRestArguments"
```

Comment:
using autorest-beta sometimes calls into global installation make sure we are always calling into local one